pkg/syncer/cnsoperator/controller/cnsnodevmattachment/cnsnodevmattachment_controller.go

```diff
@@ -621,8 +621,9 @@ func (r *ReconcileCnsNodeVMAttachment) Reconcile(ctx context.Context,
 	}
 	resp, faulttype, err := reconcileCnsNodeVMAttachmentInternal()
 
-	if (err != nil || resp != reconcile.Result{}) {
+	if err != nil || faulttype != "" {
```

Comment:
Are you ok with returning empty resp?
Can we not make sure `reconcileCnsNodeVMAttachmentInternal` always returns fault when err is not nil?